CONTRIBUTING.md

```diff
@@ -1,21 +1,129 @@
-# Contributing
+# How to contribute
 
-## Evolving the Format of the Forecast Data
```

Comment:
I would like to keep this section and the content that was in it. Could you please add it back somewhere?

Reply:
Surething, done in CONTRIBUTING.md